app-hitl/hitl/src/hitl/commands/run_hitl.py

```diff
@@ -60,13 +68,11 @@ def run_hitl(job_id):
         output_dir=output_location,
         last_output_dir=None,
         train_kw=dict(
-            num_epochs=5, chip_sz=256, img_sz=256, external_model=True),
+            num_epochs=1, chip_sz=256, img_sz=256, external_model=True),
```

Comment:
@aaronxsu do we want to toggle this back to 5 before merging? If so I'm happy to make that change

Reply:
Yes, please change it back to `5`.